### PR TITLE
update download url to use fastly

### PR DESCRIPTION
**Pull request by dwradcliffe**
(instead of old hardcoded s3 bucket)
This is already running in production via heroku config variable.

@indirect @andremedeiros @evanphx

### CODE REVIEW

**Comment by segiddins** [comment]
:+1:

File: .env [hunk 1]
```diff
@@ -11,5 +11,5 @@ MAX_THREADS=16
 REDIS_ENV=REDIS_URL
 REDIS_URL=redis://127.0.0.1:6379
 MEMCACHE_SERVERS=127.0.0.1:11211
-RUBYGEMS_HOST=http://rubygems.org
-DOWNLOAD_BASE=http://production.s3.rubygems.org
+RUBYGEMS_HOST=https://rubygems.org
```
Comment (segiddins) [inline .env]: should this be `RUBYGEMS_URL` ?

Comment (dwradcliffe) [inline .env]: Good catch; both versions were in the code. I've consolidated to use `RUBYGEMS_URL` everywhere.

**Comment by segiddins** [comment]
@homu r+

**Pull request merged**